packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1778,36 +1753,90 @@ addFieldsDict(Users, {
     canRead: ['guests'],
     canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
     form: {
-      inputPrefix: SOCIAL_MEDIA_PROFILE_FIELDS.facebookProfileURL
+      inputPrefix: SOCIAL_MEDIA_PROFILE_FIELDS.githubProfileURL
     },
     group: formGroups.socialMedia
   },
-  twitterProfileURL: {
+  
+  profileTagIds: {
+    ...arrayOfForeignKeysField({
+      idFieldName: "profileTagIds",
+      resolverName: "profileTags",
+      collectionName: "Tags",
+      type: "Tag"
+    }),
+    hidden: true,
+    optional: true,
```

Comment:
Can this be nullable (since we seem to be treating it that way in `userCallbacks`), or does that break something?  (We aren't assigned a default value to it, so I assume it'll often be missing at runtime.)

Reply:
Yup it's an optional field